hollow/src/main/java/com/netflix/hollow/api/metrics/HollowConsumerMetrics.java

```diff
@@ -18,9 +18,31 @@
 
 import com.netflix.hollow.core.read.engine.HollowReadStateEngine;
 
+import java.util.concurrent.atomic.AtomicLong;
+
 public class HollowConsumerMetrics extends HollowMetrics {
     private int refreshFailed;      // TODO: Move these metrics over to com.netflix.hollow.api.consumer.metrics.AbstractRefreshMetricsListener
     private int refreshSucceeded;
+    private final AtomicLong lastRefreshStartTimeNs = new AtomicLong(); //last time a refresh started
+    private final AtomicLong lastRefreshEndTimeNs = new AtomicLong();
+
+    public void setLastRefreshStartTimeNs() {
```

Comment:
nit: since System.nanoTime is not related to wall clock time, we can drop "Time" from the method name, so like `setLastRefreshStartNanos`.

Also nit: convention with setters and getters, setters will take the value as an argument